python/lsst/ts/ess/labjack/labjack_data_client.py

```diff
@@ -122,7 +123,7 @@ def get_config_schema(cls) -> Dict[str, Any]:
   identifier:
     description: >-
         LabJack indentifier:
-        * An IP address if connection_type=TCP
+        * An IP host name or address if connection_type = TCP or WIFI
```

Comment:
I think `host name or IP address` would be better. In networking nobody refers to `IP host name` to my best of knowledge.